### ISSUE TITLE
SPDX coverage from what FOSSology covers

### ISSUE THREAD

**Issue by mcjaeger** [spdx/spdx-spec#112]
Proposed is a meta issue that shows what FOSSology covers in terms of license analysis use cases and which of the issues in this issue tracker are connected with this.

Please note also that there is a meta issue on the FOSSology side, summarising captured shortcomings of the FOSSology w.r..t the SPDX spc:

https://github.com/fossology/fossology/issues/1079

### FOSSology use cases

- [ ] Obligations, risks, restrictions as part of the SPDX https://github.com/spdx/spdx-spec/issues/111
- [ ] ECC information
- [ ] IPR limitations could be also covered by risks risks
- [ ] identified files that do not get distributed https://github.com/spdx/spdx-spec/issues/92
- 

Admitted that #92 is maybe covered by the relations construct, although I am not convinced it represents a solution that as intended

**Comment by iamwillbar** [comment]
A lot of these feel they would fit nicely into the profiles model in 3.0. I suggest we move this issue to the 3.0 milestone.

**Comment by swinslow** [comment]
Agree, I think this is more appropriate to consider as part of 3.0.

**Comment by goneall** [comment]
This may be a good issue to add to the [operations](https://github.com/spdx/meetings/tree/main/operations) profile team.

**Comment by goneall** [comment]
Moving to 3.1 - the target for the profiles team.

**Comment by swinslow** [comment]
I've commented within the sub-issues linked from here (and/or closed them) to reflect where I think these currently stand. 

Short version, I don't think there's anything left in here that is licensing-related that hasn't either already been addressed, or else is a "wontfix" for 3.0 and going forward. I'll leave this open to the extent that parts of it may touch on other profiles.
